_data/accessibility-tests/list.yml

```diff
@@ -0,0 +1,55 @@
+title: List
+component_status: pass
+test_items:
+# Zoom/screen magnification items
+  - summary: Text retains functionality when screen is magnified.
+    summary_additional: |
+      When you zoom into 200%,
+      you can still see all list content.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.4
+# Screen reader items
+  - summary: Screen reader announces list.
+    summary_additional: |
+      When you listen to a list with a screen reader,
+      the screen reader announces that it is a list and tells you how many items are in the list.
+      For example, "List; four items."
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.6.1
+    wcag_criterion: 1.3.1
+  - summary: Screen reader announces list items in the correct order.
+    summary_additional: |
+      When you listen to a list with a screen reader,
+      the screen reader announces list items in the same order that they appear on the screen (e.g., top to bottom for English pages).
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.6.1
+    wcag_criterion: 1.3.1
+  - summary: Screen reader correctly reads unordered lists.
+    summary_additional: |
+      When you listen to an unordered list with a screen reader,
+      you hear "bullet item name" before each list item.
+    test_status: pass
+    test_type: screen reader
+    version_tested: 3.6.1
+    wcag_criterion: 1.3.1
+  - summary: Screen reader correctly reads ordered lists.
+    summary_additional: |
+      When you listen to an ordered list with a screen reader,
+      you hear the number announced before each list item (e.g., "Two; list item text").
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.6.1
+    wcag_criterion: 1.3.1
+  - summary: Screen reader announces list heading.
+    summary_additional: |
+      When you listen to a list with a screen reader,
+      you hear an appropriate heading before you hear the list.
+      For example, you might hear the heading "Government buildings" followed by an unordered list.
+    test_status: pass
```

Comment:
@amycole501 @alex-hull 

**question:** Is this saying that every list must have a header immediately preceding it in order to be accessible? I'm hesitating on this one because the way I am reading it feels surprising to me, and I want to make sure I understand.

Reply:
Curious if this falls into the category of "general layout" accessibility guidance, and less in the strictly "list" guidance. Considering a list that comes after a paragraph, I wouldn't expect to hear the heading before the list.

Reply:
I think that is a good point, Charlie. Wondering if this something we could add to some list pattern guidance, as I think it is at least best practice to ensure that there is some sort of element telling the user what the list is about. Wondering if we could generalize it and say something like, "When listening with a screen reader, appropriate markup is used to inform the user they have entered a list." Based on [this technique](https://www.w3.org/WAI/WCAG22/Techniques/html/H48.html)

Reply:
I think that makes sense! Looks like this is related to the `1.3.1` criteria which we outlined already. 

Since the [semantic markup](https://dequeuniversity.com/tips/list-markup) for lists does not include any heading tags, would it be appropriate to remove this test?

Reply:
I think it's cool to remove it! I would love in the future pointing to any guidance directly from these accessibility tests (hopes and dreams for the FAR future).

Reply:
Thanks for the conversation @mahoneycm and @alex-hull. @RachelCorsino, can you remove this item from the page?

Reply:
Resolved in [96c11ec](https://github.com/uswds/uswds-site/pull/2746/commits/96c11ec7758894a1c077bf301dd0415293d9d6e4)